main/githooks.py

```diff
@@ -107,6 +107,19 @@ def get_branch():
         return _get_output('git branch').split()[-1]
 
 
+def get_text_file_content(filename):
+    '''Get content of a text file
+
+    Locally (ie. non-github event) we return the content of the staged file,
+    not the file in the working directory.
+    '''
+    if _is_github_event():
+        data = Path(filename).read_text()
+    else:
+        data = _get_output(f'git show :{filename}')
```

Comment:
I wondered whether this would include line ending conversions?  The docs for git show suggest it is the raw content of the blob - though I guess that would be fine as it would pick up anything that hadn't converted correctly.  

What this comment really comes down to, is please test this change on some commits on both windows and linux before merging.

Reply:
We haven't changed how we read the file to check line ending, but I realize we should. I need to update the code and test again.